test/Riok.Mapperly.Tests/Mapping/UserMethodTest.cs

```diff
@@ -346,4 +411,32 @@ class B { public string Value { get; set; } public short Value2 { get; set; } }
         );
         return TestHelper.VerifyGenerator(source);
     }
+
+    [Fact]
+    public Task ExistingTargetWithMapperClassModifiersShouldCopyModifiersToMapper()
```

Comment:
I don't think this needs an extra test, the code for an existing target mapper regarding the copy of the modifiers is exactly the same.